docs/release-notes/8.12.asciidoc

```diff
@@ -0,0 +1,77 @@
+[[release-notes-header-8.12.0]]
+== 8.12
+
+[discrete]
+[[release-notes-8.12.0]]
+=== 8.12.0
+
+[discrete]
+[[known-issue-8.12.0]]
+==== Known issues
+
+There are no known issues in 8.12.0.
+
+[discrete]
+[[breaking-changes-8.12.0]]
+==== Breaking changes
+
+There are no breaking changes in 8.12.0.
+
+[discrete]
+[[deprecations-8.12.0]]
+==== Deprecations
+
+There are no deprecations in 8.12.0.
+
+[discrete]
+[[features-8.12.0]]
+==== New features
+
+* Introduces the ability to assign alerts to specific users ({pull}170579[#170579], {pull}171589[#171589]).
+* Introduces Retrieval Augmented Generation (RAG) for Alerts, allowing you to give Elastic AI Assistant context about more alerts in your environment ({pull}172542[#172542]).
+* Enables alert suppression for threshold rules ({pull}171423[#171423]).
+* Adds an *Updates* tab to the prebuilt rules upgrade flyout to show differences between the installed and updated versions ({pull}172535[#172535], {pull}173187[#173187]).
+* Adds a setting that lets you exclude cold and frozen tiers from visual event analyzer queries ({pull}172162[#172162]).
+* Adds a tour to guide users through Timelines UI changes ({pull}172030[#172030]).
+* Adds a timeout option for Osquery queries, so you can customize the maximum time each query should run before timing out ({pull}169925[#169925]).
+* Introduces new grouping capabilities for CSPM and KSPM Findings data ({pull}169884[#169884]).
+* Adds the expandable alert details flyout to the rule preview panel ({pull}167902[#167902]).
+* Adds a button to rule execution error messages that lets you ask AI Assistant to diagnose errors ({pull}166778[#166778]).
```

Comment:
We don't need this line any more, since it's also been added to Enhancements below. AND the SentinelOne bidirectional response feature is missing from this list, so we can get two birds with one commit here by just replacing the line:
```suggestion
* preview:[] Introduces bidirectional response actions to isolate and release SentinelOne-protected hosts.
```

It's a big, sprawling feature so I'm having trouble finding specific PRs to reference. The epic issue is https://github.com/elastic/security-team/issues/6200, but that's in a private repo so probably not useful to include here.

Reply:
@joepeeples do you have a preference for how we label this feature as `tech preview` in the release notes? In the past, we've taken the less formal approach of adding `(technical preview)` at the end of the release note summary to avoid interrupting the summary flow ([example](https://github.com/elastic/security-docs/blob/d7987bb5d2703ce462d849fedad669f502355d30/docs/release-notes/8.11.asciidoc?plain=1#L89)). That said, there's currently no standard for handling the tech preview label in release notes, so I have no objections to using the asciidoc tag if it's more appropriate for this case.

Reply:
No preference, thanks!

Reply:
Ok, cool. Just making a tiny edit to your edit then:

```suggestion
* Introduces bidirectional response actions to isolate and release SentinelOne-protected hosts (technical preview).
```